netci.groovy

```diff
@@ -61,33 +70,113 @@ def static getBuildJobName(def configuration, def architecture, def os) {
 
 // Adds a trigger for the PR build if one is needed.  If isFlowJob is true, then this is the
 // flow job that rolls up the build and test for non-windows OS's
-def static addPRTrigger(def job, def architecture, def os, def configuration, isFlowJob) {
+def static addTriggers(def job, def isPR, def architecture, def os, def configuration, def scenario, def isFlowJob) {
+    // Non pull request builds.
+    if (!isPR) {
+        // Check scenario.
+        switch (scenario) {
+            case 'default':
+                // default gets a push trigger for everything
+                Utilities.addGithubPushTrigger(job)
+                break
+            case 'pri1':
+                // Pri one gets a daily build, and only for release
+                if (architecture == 'x64' && configuration == 'Release') {
+                    // We don't expect to see a job generated except in these scenarios
+                    assert os == 'Windows_NT' || isFlowJob && os in crossList
```

Comment:
Should it be something like  os == 'Windows_NT' || (isFlowJob && os in crossList) ?